aiida/cmdline/commands/profile.py

```diff
@@ -75,13 +75,21 @@ def profile_setdefault(self, *args):
 
     def profile_list(self, *args):
         from aiida.common.setup import get_profiles_list, get_default_profile, AIIDA_CONFIG_FOLDER
+        from aiida.common.exceptions import ConfigurationError
+
         from aiida.backends import settings
 
         print('Configuration folder: {}'.format(AIIDA_CONFIG_FOLDER))
 
         current_profile = settings.AIIDADB_PROFILE
-        default_profile = get_default_profile(
+        try:
+            default_profile = get_default_profile(
                 settings.CURRENT_AIIDADB_PROCESS)
+        except ConfigurationError as e:
+            print >> sys.stderr, "Stopping: {}".format(e.message)
```

Comment:
since we already import `click` on top of the file, let's use
```python
click.echo(<message>, err=True)
```
instead of the `print` statement.

Reasoning:
* consistency
* python3 compatible
* more readable (IMO)